### PR TITLE
chore: eslint fixes

### PR DESCRIPTION
**Pull request by tibuurcio**
## Summary
ESLint fixes

## Testing Plan
- [X] Was this tested locally? If not, explain why.

### CODE REVIEW

File: src/components/navigation/GlobalNavigation/NavigationList.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ function generateMenuItem(item: IGlobalNavigationItem, i: number) {
     const childrenWithExpandedIcons = item.children.map((child, index) => ({
       ...child,
       expandIcon: null,
-      key: `${child.label}${index}`,
+      key: `${String(child.label)}${index}`,
```
Comment (jared-dickman) [inline src/components/navigation/GlobalNavigation/NavigationList.tsx:40]: what is this, java?!

File: src/components/general/Icon/Icon.stories.tsx [hunk 2]
```diff
@@ -15,7 +13,7 @@ export const IconTable: React.FC<IIconProps> = ({ color = 'black', size = 'lg' }
       justifyItems: 'center',
     }}>
     {(Object.keys(Icons) as Array<keyof typeof Icons>).map(iconName => (
-      <Flex vertical align="center">
+      <Flex vertical align="center" key={iconName}>
```
Comment (jared-dickman) [inline src/components/general/Icon/Icon.stories.tsx:16]: makes me think, should we add a `key` property to the Icon component for example

**Review by jared-dickman — APPROVED** [review]

File: src/components/navigation/GlobalNavigation/NavigationList.tsx (hunk 1 above)
Comment (tibuurcio) [reply src/components/navigation/GlobalNavigation/NavigationList.tsx:40]: 😆😆 I though about typechecking but it was sooo verbose that I just did this

File: src/components/general/Icon/Icon.stories.tsx (hunk 2 above)
Comment (tibuurcio) [reply src/components/general/Icon/Icon.stories.tsx:16]: Don't think so 🤔 
Icon should be a one off thing and whoever decides to use it as a list should define the key for it. Also, I think all React c omponents already accept a key right?

**Pull request merged**